src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsPage.js

```diff
@@ -35,11 +35,12 @@ const propTypes = {
     iou: iouPropTypes,
 
     /** The current tab we have navigated to in the request modal. String that corresponds to the request type. */
-    selectedTab: PropTypes.oneOf([CONST.TAB.DISTANCE, CONST.TAB.MANUAL, CONST.TAB.SCAN]).isRequired,
+    selectedTab: PropTypes.oneOf([CONST.TAB.DISTANCE, CONST.TAB.MANUAL, CONST.TAB.SCAN]),
 };
 
 const defaultProps = {
     iou: iouDefaultProps,
+    selectedTab: '',
```

Comment:
This results in another error
> Failed prop type: Invalid prop `selectedTab` of value `` supplied to `MoneyRequestParticipantsPage`, expected one of ["distance","manual","scan"].

I think it should be `selectedTab: undefined,`

Reply:
Fixed it up.